.circleci/config.yml

```diff
@@ -24,7 +22,7 @@ canary_tests: &canary_tests
         command: npm i --no-save webpack@next
     - run:
         name: Run unit tests.
-        command: npm run ci:test
+        command: if [[ $(compver --name webpack --gte next --lt latest) < 1 ]] ; then printf "Next is older than Latest - Skipping Canary Suite"; else npm run ci:test ; fi
```

Comment:
If I was going to quibble I'd probably say this is long enough to be worth using a multiline string now - but not going to block a merge for that.